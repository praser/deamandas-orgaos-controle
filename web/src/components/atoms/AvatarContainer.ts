import styled from 'styled-components';

const AvatarContainer = styled.div`
  /* layout */
  box-sizing: border-box;
  display: block;
  height: 32px;
  width: 32px;

  /* appearence */
  border-style: none;
  color: rgb(209, 211, 226);
  cursor: pointer;
  overflow: hidden;

  /* other */
  border-radius: 50%;
  list-style-image: none;
  list-style-position: outside;
  list-style-type: none;
  -webkit-box-direction: normal;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
`;

export default AvatarContainer;
