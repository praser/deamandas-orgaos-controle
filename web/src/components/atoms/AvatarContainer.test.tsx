import React from 'react';
import { render, screen } from '@testing-library/react';
import AvatarContainer from './AvatarContainer';

describe('Avatar container atomic component', () => {
  let avatarContainer: HTMLElement;

  beforeEach(() => {
    render(<AvatarContainer data-testid="container" />);
    avatarContainer = screen.getByTestId('container');
  });

  it('is expected to match the snapshot', () => {
    expect(avatarContainer).toMatchSnapshot();
  });
});
