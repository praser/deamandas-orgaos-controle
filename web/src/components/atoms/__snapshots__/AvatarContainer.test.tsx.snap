// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar container atomic component is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa kEjUpP"
  data-testid="container"
/>
`;
